### DISCUSSION TITLE
apple bug

### DISCUSSION THREAD

**Discussion by hiimdemil** [CryptechTest/cryptech#128] [Ideas]
There is a kind of bug in the apples, once you cut the tree the apples don't fall to the ground, nor do they disappear... the apples can stay in the air indefinitely.
![image](https://github.com/CryptechTest/cryptech/assets/125774760/8a20603c-aca1-4fd6-9b31-f47252dfa444)

**Comment by squidicuzz** [comment]
Corrected this.  Thanks for the report!
